src/foam/nanos/fs/File.js

```diff
@@ -267,7 +279,8 @@ foam.CLASS({
       name: 'authorizeOnDelete',
       javaCode: `
         AuthService auth = (AuthService) x.get("auth");
-        if ( ! auth.check(x, "file.remove." + getId()) ) {
+
+        if ( ! auth.check(x, "file.remove.*") ) {
```

Comment:
@Mykola97 @thepinkcacao  why not "file.remove.id"?

Reply:
@lchanmann  does id mean "fileID" or "UserID"?